### ISSUE TITLE
Failed to upload the lms.zip to s3 bucket

### ISSUE THREAD

**Issue by KageYang** [aws-samples/amazon-chime-sdk-classroom-demo#84]
Hi there :

The deployment failed when uploading the lms.zip to s3 bucket. It mentioned "release/lms.zip does not exist. 
Please kindly advise. Thanks a lot !

... uploading Mac installer (this may take a while) ...
--> aws s3 cp --acl public-read release/lms.zip s3://chime-211259373114-releases/mac/lms.zip

Command aws failed with exit code 255 signal null

The user-provided path release/lms.zip does not exist.

**Comment by jing-chen1** [comment]
Closing it, feel free to reopen it if you still have issues

**Comment by rajleechaudry** [comment]
I am experiencing the same problem.  The error is saying the mac zip file didn't get created, and I don't see the mac zip file in the release directory.  So I postulate the problem is the mac zip file is not being created.  Yet, I don't see an error either.  This is on Cloud9.  You might want to reopen this, because I think something is wonky with the deploy script.

... uploading Mac installer (this may take a while) ...
--> aws s3 cp --acl public-read release/human0.zip s3://human0-releases/mac/human0.zip

Command aws failed with exit code 255 signal null

The user-provided path release/human0.zip does not exist.

**Comment by DjWilly** [comment]
I'm on the same boat.. Getting this same error over and over again. I tried to upper the size to 2TB and it still complaining :(

btw, the tutorials out there just assume that's you have setup everything perfectly to work with Chime!
I've spend the whole week trying to get this classroom to work. Any direction to a good tutorial please?
Ideally with a little background on how to prepare aws before we start typing in those deploy commands.

**Comment by nainkunal933** [comment]
@DjWilly Sorry you are facing an issue with setting up the demo. I think we missed your comment since this ticket was closed. I am not sure if you have resolved your issue. If not, can you please give me some information on deployment option you are using? In future, if you ever have any issues or questions, please feel free to open a new GitHub issue. That will ensure someone is able to take a look at your issue. We track open issues that we have not replied back to but closed issues are not tracked.

I just ran the classroom setup demo in Cloud9 and faced an issue with the mac zipped files. I am currently looking into option 1 and seeing if I can push a quick patch. I also tried deploying the demo via option 2 and can confirm that option 2 works.

I will reopen this issue to track the setup bug.

**Comment by nainkunal933** [comment]
For now, the mitigation strategy will be to use option 2. Please let me know if you run into any issues there.

**Comment by nainkunal933** [comment]
I have merged in a fix that should help with the S3 copy issue in the deploy script. Please feel free to open another issue if you still face an issue.
